### ISSUE TITLE
To allow to switch between dark and light theme

### ISSUE THREAD

**Issue by sallymoc** [qubic/wallet-app#73]
From product-feedback channel https://discord.com/channels/768887649540243497/1257429576313864323/1257467648405536880

**Comment by sallymoc** [comment]
@Orestasas  do you have any ideas on this?

**Comment by Orestasas** [comment]
@sallymoc yeah, if we are thinking of having it, I can plan for it. I see it as a lower priority item that could be done after we do all pending UI improvements
